CHANGELOG.latest.md

```diff
@@ -1,30 +1,35 @@
-## 5.8.2
+#### Introducing Signature Verification (beta):
 
-⚠️ ⚠️ ⚠️ ⚠️ 
+This new feature prevents MitM attacks between the SDK and the RevenueCat server.
+With verification enabled, the SDK ensures that the response created by the server was not modified by a third-party, and the entitlements received are exactly what was sent.
+This is 100% opt-in. `EntitlementInfos` and `EntitlementInfo` have a new `VerificationResult` property, which will indicate the validity of the responses when this feature is enabled.
 
-Our Android SDK versions 5.8.0 and 5.8.1 introduced a bug that prevented new purchase tokens from being sent to RevenueCat for validation between the above dates.
+```kotlin
+val purchasesConfiguration = PurchasesConfiguration.Builder(application, apiKey)
+	.entitlementVerificationMode(EntitlementVerificationMode.informational)
+	.build()
+val purchases = Purchases.configure(purchasesConfiguration)
 
-Users who made new purchases were charged by Google but did not receive entitlements during that time. We've already shipped a backend fix, so affected users can recover their purchases simply by opening the app again. If the purchases aren’t recovered by the users within 72 hours from their purchase date, Google will automatically refund their purchase. No further action is required from you at this time.
+purchases.getCustomerInfoWith(
+	onError = {...},
+	onSuccess = { customerInfo -> 
+		if (customerInfo.entitlements.verification != VerificationResult.SUCCESS) {
+			println("Entitlements could not be verified")
+		}
+	}
+)
+```
 
-Users with pre-existing purchases are not affected.
+#### Grace period for entitlements
```

Comment:
@NachoSoto @aboedo I added this section to the changelog. I think it deserves to be called out prominently as well, since it's an important behavior change compared with previous versions. Let me know if you have any thoughts!